src/partials/messenger.ts

```diff
@@ -1267,6 +1267,21 @@ class NavigationController {
         this.searchVisible = !this.searchVisible;
     }
 
+    /**
+     * Toggle search bar off.
+     */
+    public toggleSearchOff(): void {
+        if (this.searchVisible) this.searchVisible = false;
```

Comment:
Nitpick: Can you add explicit curly braces? We prefer that for explicitness.

```suggestion
        if (this.searchVisible) {
            this.searchVisible = false;
        }
```